[2024-01-05T10:21:01.730+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1 permission to 509
[2024-01-05T10:21:01.731+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1'
[2024-01-05T10:21:01.764+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1 permission to 509
[2024-01-05T10:21:01.765+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1'
[2024-01-05T10:21:01.801+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: User_counter.TaskId1 scheduled__2024-01-05T10:20:00+00:00 [queued]>
[2024-01-05T10:21:01.810+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: User_counter.TaskId1 scheduled__2024-01-05T10:20:00+00:00 [queued]>
[2024-01-05T10:21:01.811+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2024-01-05T10:21:01.828+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): TaskId1> on 2024-01-05 10:20:00+00:00
[2024-01-05T10:21:01.834+0000] {standard_task_runner.py:60} INFO - Started process 3574 to run task
[2024-01-05T10:21:01.839+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'User_counter', 'TaskId1', 'scheduled__2024-01-05T10:20:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/User_counter.py', '--cfg-path', '/tmp/tmpmnpj11nz']
[2024-01-05T10:21:01.843+0000] {standard_task_runner.py:88} INFO - Job 11: Subtask TaskId1
[2024-01-05T10:21:01.898+0000] {logging_mixin.py:188} INFO - Changing /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1 permission to 509
[2024-01-05T10:21:01.899+0000] {logging_mixin.py:188} INFO - Failed to change /opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1 permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=User_counter/run_id=scheduled__2024-01-05T10:20:00+00:00/task_id=TaskId1'
[2024-01-05T10:21:01.902+0000] {task_command.py:423} INFO - Running <TaskInstance: User_counter.TaskId1 scheduled__2024-01-05T10:20:00+00:00 [running]> on host 098ababd5647
[2024-01-05T10:21:02.000+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='User_counter' AIRFLOW_CTX_TASK_ID='TaskId1' AIRFLOW_CTX_EXECUTION_DATE='2024-01-05T10:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-05T10:20:00+00:00'
[2024-01-05T10:21:02.003+0000] {logging_mixin.py:188} INFO - ZAEBALLL11
[2024-01-05T10:21:02.003+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-05T10:21:02.015+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=User_counter, task_id=TaskId1, execution_date=20240105T102000, start_date=20240105T102101, end_date=20240105T102102
[2024-01-05T10:21:02.051+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-05T10:21:02.080+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
